spec/features/search/search_spec.rb

```diff
@@ -1,6 +1,16 @@
 require "rails_helper"
 
 describe "Search page", type: :feature do
+  it "translates the close button to German" do
+    page.driver.header 'Accept-language', 'de-DE'
+    visit search_path
+    expect(page).to have_text("Schließen")
+  end
+
+  it "translates the close button to English by default" do
+    visit search_path
+    expect(page).to have_text("Close")
+  end
```

Comment:
Tests are generally great, but do you want to do it this way anywhere? Another idea would be verifying that the t-function contains the needed keys. But what's better? What did the other team do?